src/Pos/Update/Poll/PollState.hs

```diff
@@ -0,0 +1,77 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+-- Types related Poll update system
+
+module Pos.Update.Poll.PollState
+       (
+         -- * Types
+         PollState (..)
+
+       -- * Lenses
+       -- * PollState
+       , psBlockVersions
+       , psAdoptedBV
+       , psSoftwareVersion
+       , psEpochProposers
+       , psConfirmedBVs
+       , psConfirmedProposals
+       , psActiveProposals
+       , psDelActivePropsIdx
+       , psSlottingData
+       , psUndecidedProposals
+       , psDecidedProposals
+       , psStakePerEpoch
+       , psMultiRichmenStake
+       , psIssuersStakes
+       ) where
+
+import           Control.Lens                 (makeLenses)
+import           Data.HashSet                 (HashSet)
+import           Data.HashMap.Strict          (HashMap)
+import           Universum
+
+import           Pos.Core.Types               (BlockVersionData, Coin)
+import           Pos.Lrc.DB.Issuers           (IssuersStakes)
+import           Pos.Slotting.Types           (SlottingData)
+import           Pos.Ssc.GodTossing.Toss.Pure (MultiRichmenStake)
```

Comment:
Ok, it actually makes sense, but then it also makes sense to move this alias to `Pos.Lrc.Types` which is in `lrc` package to avoid dependency on `Ssc`.

Anyway, I see that you solved it slightly differently, ok.